config/ymodel/sidebar_items.yml

```diff
@@ -4,1145 +4,362 @@
   percentage_bar_query: share_of_households_in_final_demand
   tab_key: demand
   position: 1
-  dependent_on:
-  parent_key:
-  description:
-    content_en: "How much energy will households need in the future?\r\n<br/>\r\n<br/>\r\n\r\nThe
-      answer depends on your assumptions about the adoption of new technologies and
-      economic or population growth, for example. You can fill in these assumptions
-      and the graphs will show how things change. \r\n<br/>\r\n<br/>\r\nHouseholds
-      mostly need heat and electricity. Heat use has been steadily decreasing the
-      past years, because of better insulation in houses. Electricity demand has kept
-      increasing. As people get richer, they use ever more electrical appliances.\r\n\r\n\r\n"
-    content_nl: "Hoeveel energie zullen huishoudens nodig hebben in de toekomst? \r\n<br/><br/>\r\n\r\nDat
-      hangt af van je aannames over de toepassing van nieuwe technologieën en economische
-      of bevolkingsgroei bijvoorbeeld. Deze aannames kun je hier invullen. In de grafieken
-      zie je dan hoe alles verandert. \r\n<br/><br/>\r\nHuishoudens hebben vooral
-      behoefte aan warmte en elektriciteit. De warmtevraag neemt gestaag af door betere
-      isolatie van gebouwen. De elektriciteitsvraag is alsmaar gegroeid. Als mensen
-      rijker worden, gaan ze meer elektrische apparaten gebruiken.\r\n\r\n"
-    short_content_en: "How much energy will households need in the future?\r\nThe
-      answer depends ..."
-    short_content_nl: Hoeveel energie zullen huishoudens nodig hebben in de toekomst?
-      Dat hangt ervan af ...
+  dependent_on: 
```

Comment:
Could you remoe the trailing whitespace?

Reply:
Yes! I guess I forgot that one, good catch!